nixos/tests/public-inbox.nix

```diff
@@ -10,7 +10,6 @@ import ./make-test-python.nix (
       install -D -t $out key.pem cert.pem
     '';
 
-    # Git repositories paths in Gitolite.
```

Comment:
Is this removal intentional?

Reply:
Ah okay, thanks! Should probably be a separate commit since it's not related to testing confinement.